files/es/glossary/mixin/index.md

```diff
@@ -13,7 +13,7 @@ width: 100% ;
 
 }
 
-La forma más común de llamar a un mixin y aplicar esas propiedades y métodos es el _include_ (ejemplo @include nombre_del_mixin ). Al utilizar el _include_ para llamar las propiedades y métodos incluidos en el mixin, son entonces parte de la nueva interfaz.
+La forma más común de llamar a un mixin y aplicar esas propiedades y métodos es el _include_ (ejemplo @include nombre*del_mixin ). Al utilizar el \_include* para llamar las propiedades y métodos incluidos en el mixin, son entonces parte de la nueva interfaz.
```

Comment:
Seems that it missunderstood the unserscore

```suggestion
La forma más común de llamar a un mixin y aplicar esas propiedades y métodos es el _include_ (ejemplo `@include nombre_del_mixin`). Al utilizar el _include_ para llamar las propiedades y métodos incluidos en el mixin, son entonces parte de la nueva interfaz.
```

Reply:
Applying this suggestion, but we should probably delete this file as it is _very_ out-of-sync with upstream.  WDYT?